icons/picard.svg

```diff
@@ -1,161 +1,142 @@
-  <path clip-rule="evenodd" d="M 179.24718,697.80235 L 179.8333,700.95475 L 216.65884,893.68364 L 217.30046,896.86314 C 226.38982,944.44334 244.89881,959.40954 286.8713,952.69244 L 835.75776,865.10744 C 883.78126,857.44073 897.33316,835.98252 886.34866,784.87805 L 885.60366,781.43893 L 817.92926,467.35118 L 817.21366,463.98616 L 793.40016,353.56172 L 754.83986,174.52094 L 731.53156,66.373589 C 721.10776,18.021029 700.58596,4.7266292 654.93806,16.451239 L 132.00564,150.91292 C 91.902436,161.2012 80.566606,181.3855 89.202376,226.59461 L 108.52765,327.65919 L 159.43133,594.05014 L 179.24718,697.80235 z M 362.59378,208.12246 C 348.02258,181.85942 360.89478,152.30769 391.28605,142.07361 C 421.69222,131.8765 458.12088,144.86419 472.69209,171.12724 C 487.26335,197.39029 474.39109,226.94202 443.99972,237.17619 C 413.59371,247.37327 377.16514,234.3855 362.59378,208.12246 z" id="path3357" style="opacity:0.45;fill:#000000;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:4.06873464999999970;stroke-linecap:round;stroke-linejoin:round;stroke-opacity:1;filter:url(#filter3387)"/>
-  <path clip-rule="evenodd" d="M 166.01314,707.80235 L 166.59926,710.95475 L 203.4248,903.68363 L 204.06642,906.86316 C 213.15578,954.44329 231.66477,969.40949 273.63726,962.69249 L 822.52374,875.10747 C 870.54724,867.44073 884.09914,845.98252 873.11464,794.87805 L 872.36954,791.43893 L 804.69524,477.35118 L 803.97964,473.98616 L 780.16614,363.56172 L 741.60574,184.52094 L 718.29754,76.373589 C 707.87364,28.021029 687.35194,14.726629 641.70404,26.451239 L 118.7716,160.91292 C 78.668396,171.2012 67.332566,191.3855 75.968336,236.59461 L 95.293611,337.65919 L 146.19729,604.05014 L 166.01314,707.80235 z M 349.35974,218.12246 C 334.78854,191.85942 347.66074,162.30769 378.05201,152.07361 C 408.45818,141.8765 444.88684,154.86419 459.45804,181.12724 C 474.02934,207.39029 461.15704,236.94202 430.76568,247.17619 C 400.35967,257.37327 363.9311,244.3855 349.35974,218.12246 z" id="path6464" style="opacity:0.66535429000000001;fill:#000000;fill-opacity:1;fill-rule:eveno
```

Comment:
Could you please remove the width and height here?
I know it is part of the official icon, but the only purpose of this is to force a a maximum width and height of the svg icon, and shouldn't really be necessary.

Reply:
Since it shows on the wrong line in the review comment, I am talking about removing line 16 and 17 in the new icon svg.

Reply:
Sure, makes sense. I updated it with the width and height removed.